README.md

```diff
@@ -98,6 +98,7 @@ set -g theme_avoid_ambiguous_glyphs yes
 set -g theme_powerline_fonts no
 set -g theme_nerd_fonts yes
 set -g theme_show_exit_status yes
+set -g theme_show_job_number yes
```

Comment:
`theme_display_jobs_verbose` would probably be more consistent with the other configuration options. `theme_show_exit_status` is the outlier :P

Reply:
Will do. Didn't know that `theme_show_exit_status` was the outlier :D